lib/core/Canvas.js

```diff
@@ -775,7 +776,7 @@ Canvas.prototype.removeRootElement = function(rootElement) {
 Canvas.prototype.setRootElement = function(rootElement) {
 
   if (rootElement === this._rootElement) {
-    return;
+    return rootElement;
```

Comment:
Great find. Could you [add a test case](https://github.com/bpmn-io/diagram-js/blob/develop/test/spec/core/CanvasSpec.js#L584) for this?

Reply:
Added a test for this https://github.com/bpmn-io/diagram-js/pull/875/files#diff-11db5d0fde9080dce33b2892042ad3e3913973a2bc59a46d784d592041df3e07R600-R611